src/lando/jinja2.py

```diff
@@ -323,15 +336,7 @@ def environment(**options):
             "graph_height": graph_height,
             "graph_width": graph_width,
             "graph_x_pos": graph_x_pos,
-            "is_user_authenticated": is_user_authenticated(env),
-            "linkify_bug_numbers": linkify_bug_numbers,
-            "linkify_faq": linkify_faq,
-            "linkify_revision_ids": linkify_revision_ids,
-            "linkify_revision_urls": linkify_revision_urls,
-            "linkify_sec_bug_docs": linkify_sec_bug_docs,
-            "linkify_transplant_details": linkify_transplant_details,
```

Comment:
Why were these removed? Should they have been moved as well?

Reply:
`is_user_authenticated` was indeed moved, though in the future it will also become obsolete since we will use Django's auth functionality.

I didn't mean to remove the `linkify_` filters in this PR, will put them back!